cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -15,49 +15,19 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
```

Comment:
Line 13: `:FeatureName: {CNVProductName}` needs to be changed to `:FeatureName: {CNVProductNameStart}`